lib/config/options/index.ts

```diff
@@ -435,6 +435,13 @@ const options: RenovateOptions[] = [
     type: 'boolean',
     default: false,
   },
+  {
+    name: 'dependencyDashboardDetectedDeps',
```

Comment:
I'm not sure we need to make this configurable, unless we're uncertain about it being unstable

Reply:
Should be stable as we are just doing some string manipulation on a copy of a `dto` created at the `extraction` phase and which is updated during `lookup`.

should i drop this feature flag all together, or change the flag default to `true` as an alternative (would anyone want to turn it off? i really dont know)?

Reply:
I think let's drop the feature flag. We already have a *lot* of config options

Reply:
feature flag removed